### ISSUE TITLE
Add option to predictInterval to return only fixed or random component

### ISSUE THREAD

**Issue by jknowles** [jknowles/merTools#51]
This would allow the user to build prediction intervals on fixed, random, residual variance, or all and that would allow for comparisons about the variability in predictions that is coming from the fixed or random component of the model for particular observations.

**Comment by jknowles** [comment]
Here's some code to do only the fixed part:

```
PredDat <- data.frame(Intercept = 1, 
           ELEVATION = seq(-1.97056, 2.52926,length.out = 1000), 
           SLOPE = 0, ET = 0, HLI = 0)

 fe.tmp <- fixef(mod)
 vcov.tmp <- as.matrix(vcov(mod))
 n.sims <- 1000
 sigmahat <- rep(1, n.sims)

 # Make n.sims draws for each element of the fixed effects

 betaSim <- abind::abind(lapply(1:n.sims,
  function(x) mvtnorm::rmvnorm(n = 1, mean = fe.tmp, 
       sigma = sigmahat[x]*vcov.tmp, method = "chol")), along=1)
# Calculate n.sims predictions for each row in PredDat
fixed <- as.matrix(PredDat) %*% t(betaSim)
# For each row (observation) in PredDat calculate the median, upr and lwr 
Preds <- data.frame(fit = apply(fixed, 1, median), 
                upr = apply(fixed, 1, quantile, 0.9), 
                lwr = apply(fixed, 1, quantile, 0.1))
# Calculate the probability from the linear predictor
Preds <- apply(Preds, 2, invlogit)
````

**Comment by carlbfrederick** [comment]
Would it be possible to just capture each part as it is calculated with an if block and return any/all parts requested?
